app/src/test/java/org/simple/clinic/introvideoscreen/IntroVideoEffectHandlerTest.kt

```diff
@@ -0,0 +1,39 @@
+package org.simple.clinic.introvideoscreen
+
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import com.nhaarman.mockitokotlin2.verifyNoMoreInteractions
+import org.junit.After
+import org.junit.Test
+import org.simple.clinic.mobius.EffectHandlerTestCase
+import org.simple.clinic.util.scheduler.SchedulersProvider
+
+class IntroVideoEffectHandlerTest {
+
+  private val uiActions = mock<UiActions>()
+  private val schedulersProvider = mock<SchedulersProvider>()
```

Comment:
We should not be mocking schedulers in the tests. Instead we should be using the `TrampolineSchedulersProvider` in them. Can you please replace this mock?